handler.ts

```diff
@@ -57,6 +56,27 @@ Bugsnag.start({
 // For reference, see https://docs.bugsnag.com/platforms/javascript/aws-lambda/#usage
 const bugsnagHandler = Bugsnag.getPlugin('awsLambda').createHandler()
 
+const callPrimaryGeocoder = (apiMethod: string, event: ServerlessEvent) => {
+  // Query parameters are returned in a strange format, so have to be converted
+  // to URL parameters before being converted to a string
+  const query = new URLSearchParams(event.queryStringParameters).toString()
+
+  if (GEOCODER === 'HERE') {
+    return fetchHere(apiMethod, event.queryStringParameters, GEOCODER_API_KEY)
+  } else if (GEOCODER === 'GEOCODEEARTH') {
```

Comment:
I would rename `GEOCODEEARTH` to `PELIAS` since geocode.earth is just running a pelias instance